doc/whats_new/v0.24.rst

```diff
@@ -114,6 +114,16 @@ Changelog
   argument `rotation`, which can take the value `None`, `'varimax'` or `'quartimax'.`
   :pr:`11064` by :user:`Jona Sassenhagen <jona-sassenhagen>`.
 
+- |Fix| Avoid overflows on Windows in :func:`decomposition.IncrementalPCA.partial_fit`
+  for large ``batch_size`` and ``n_samples`` values.`
+  :pr:`17985` by :user:`Alan Butler <aldee153>` and :user:`Amanda Dsouza <amy12xx>`.
```

Comment:
This was already present on the `master` branch

Reply:
This was in 0.23 and not 0.24.